landavailability/api/management/commands/update_locations_nearest_bus.py

```diff
@@ -0,0 +1,14 @@
+from django.core.management.base import BaseCommand, CommandError
+from api.models import BusStop
+
+
+class Command(BaseCommand):
+    help = ''
+
+    def handle(self, *args, **options):
+        bss = BusStop.objects.all()
+        total_busses = len(bss)
```

Comment:
Wouldn't BusStop.objects.count() be a better solution here, rather than forcing bss into a list so we can call len()?